js/babel_get_defaults.js

```diff
@@ -0,0 +1,26 @@
+/* Systematic's babel base config file */
+// see https://github.com/babel/babel/blob/master/babel.config.js
+
+'use strict'
+
+module.exports = function () {
+  return {
+    'plugins': [
+      'transform-object-rest-spread',
+    ],
+    'env': {
+      'test': {
+        'plugins': [
+          'transform-es2015-modules-commonjs',
+          'dynamic-import-node',
+        ],
+        'presets': [
+          [
+            'env',
+            { 'targets': { 'node': 'current' } },
+          ],
+        ],
+      },
+    },
+  }
+}
```

Comment:
this does duplicate quite a bit the already present Babel config that we have in the webpack.conf.Js. It's not a bit issue though since we don't **have** to have the same conf in the tests and the compilation phases with Jest (we dont't event test in a browser, so why bother !) So... OK !